robin/src/main/java/com/sirvar/robin/LoginFragment.java

```diff
@@ -184,13 +174,115 @@ protected void showSocialLogin(boolean showLogin) {
     }
 
     private boolean fieldsFilled() {
-        return !(email.getText().toString().isEmpty() || password.getText().toString().isEmpty());
+
+
+        for( int i = 0; i < layout.getChildCount(); i++ ) {
```

Comment:
Please fix the formatting here to match the rest of the code base. Open brace on same line and fix spacing on `for( int i = 0; i < layout.getChildCount(); i++ ) {`